README.adoc

```diff
@@ -178,16 +153,15 @@ Create the `InventoryManager` class in the
 include::finish/src/main/java/io/openliberty/guides/inventory/InventoryManager.java[tags=add_fallback]
 ----
 
-The `@Fallback` annotation dictates which method to call when the automated retries are unsuccessful. In this
+The `@Fallback` annotation dictates a method to call when encounters a failed execution. In this
```

Comment:
"The `@Fallback` annotation dictates [the] method to call when [it?] encounters a failed execution."
This sentence sounds like it's missing a word. Please specify what is encountering the failed execution. Is it the annotation? the method?

Reply:
Fixed to be "The `@Fallback` annotation dictates a method to call when the original method encounters a failed execution."